Firmware/mmu2_reporting.cpp

```diff
@@ -216,11 +218,22 @@ enum ReportErrorHookStates ReportErrorHookState = ReportErrorHookStates::RENDER_
 // Helper variable to monitor knob in MMU error screen in blocking functions e.g. manage_response
 static bool is_mmu_error_monitor_active;
 
+// Helper variable to stop rendering the error screen when the firmware is rendering complementary
+// UI to resolve the error screen, for example tuning Idler Stallguard Threshold
+// Set to false to allow the error screen to render again.
+static bool putErrorScreenToSleep;
+
 bool isErrorScreenRunning() {
     return is_mmu_error_monitor_active;
 }
 
+bool isErrorScreenSleeping() {
```

Comment:
May be we could call it "TuneMenuEntered"

Reply:
Done